tests/helpers/auth-identities.ts

```diff
@@ -0,0 +1,38 @@
+import { Page } from "@playwright/test";
+import { expect } from "../fixtures/lxd-test";
+
+export const identityBar = "bar@bar.com";
+export const identityFoo = "foo@foo.com";
+
+export const visitIdentitiesPage = async (page: Page) => {
+  await page.goto("/ui/");
+  await page.getByRole("button", { name: "Permissions" }).click();
+  await page.getByRole("link", { name: "Identities" }).click();
+  await expect(
+    page
+      .getByRole("heading", { name: "Identities Learn more about" })
+      .locator("div"),
+  ).toBeVisible();
+};
+
+export const toggleGroups = async (page: Page, groups: string[]) => {
+  for (const group of groups) {
+    await page
+      .getByRole("rowheader", { name: `Select ${group}` })
+      .locator("label")
+      .click();
+    const rowModifiedIcon = page
+      .getByRole("row", { name: `Select ${group}` })
+      .locator("i");
+    await expect(rowModifiedIcon).toBeVisible();
+  }
+};
+
+export const selectIdentities = async (page: Page, identities: string[]) => {
```

Comment:
There is only a slight difference in naming this vs the `toggleIdentities` in the other file. Can we find two names that are less ambiguous?

Same with the `toggleGroups` and `selectGroups`, which are named and used in the opposite way.

Reply:
agreed, in `auth-identities.ts`, `selectIdentities` should become `selectIdentitiesToModify` and toggleGroups should become `toggleGroupsForIdentities`. Same for the equivalent helpers in `auth-groups.ts`.